tests/test_utils.py

```diff
@@ -2,7 +2,13 @@
 import pytest
 from eth_utils import keccak
 
-from raiden_libs.utils import compute_merkle_tree, get_merkle_root, EMPTY_MERKLE_ROOT
+from raiden_libs.utils import (
+    compute_merkle_tree,
+    get_merkle_root,
+    EMPTY_MERKLE_ROOT,
```

Comment:
instead of pulling a variable from the library and comparing it I think it would be just a little nicer to use something like `is_empty_merkle_root()`